### PR TITLE
Full website

### PR DESCRIPTION
**Pull request by adeola003**
In order to complete this project, I installed react-router in my project. Then I created **navbar** and **home** components.
This being done, I imported react-router components (BrowserRouter, Routes, Route) in the main component **_App.js_** and used them to refactor the content.
Lastly, I fixed the linters and tested the app.

### CODE REVIEW

File: src/components/Navbar.js [hunk 1]
```diff
@@ -0,0 +1,13 @@
+const Navbar = () => (
+  <nav className="navbar">
+    <h1>Math Magician app</h1>
+    <div className="links-container">
+      <a className="links" href="/home">Home</a>
```
Comment (Zuhaib042) [inline src/components/Navbar.js]: - [ ] Here consider changing it like this ti fix the bug `<a className="links" href="/">Home</a>`

File: src/App.js [hunk 2]
```diff
@@ -14,8 +17,14 @@ class App extends React.Component {
     return (
       <div className="App">
         <div className="App-body" id="App-body">
-          <QuoteComponent />
-          <Calculator />
+          <BrowserRouter>
+          <Navbar />
+          <Routes>
+            <Route path='/home' element={<Home />} />
```
Comment (Zuhaib042) [inline src/App.js]: - [ ] When I run the app in the browser, the route is `/` without 'home' so no page is opened. then I have to click on either link to get that specific page opened. I think when the app runs first time, the home page must get opened and displayed on the screen. for that purpose, you can write this line like this `<Route path='/' element={<Home />} />` and for it to be implemented perfectly you also need to change something in Navbar.js file. kindly have a look there for guidelines.

**Review by Zuhaib042 — CHANGES_REQUESTED** [review]
Hi @adeola003 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


## Highlights
✅ Professional PR
✅ Website is working perfectly

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by AdnanOlarmmi — APPROVED** [review]
Hi @adeola003,

This is Adnan Is-haq, your beloved code reviewer, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Approved office man

## Highlights
- Fixed reviewer's changes.

## Optional suggestions
_Check the comments under the review._

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**